### PR TITLE
BugFix: Include Model in FBX Loader in Houdini

### PR DESCRIPTION
**Pull request by MustafaJafar**
## Changelog Description
A quick bugfig where we can't load fbx exported from blender. The bug was reported [here](https://community.ynput.io/t/ayon-monthly-beginner-s-topic-january-24/1134/14).

## Additional info
You'd need to enable FBX extractor in blender `ayon+settings://blender/publish/ExtractFBX`
image

## Demo

https://github.com/ynput/OpenPype/assets/20871534/740a9d6c-88ee-461c-8df1-128379226e9f

## Testing notes:
1. export model from blender
2. load it as fbx in Houdini

### CODE REVIEW

File: openpype/hosts/houdini/plugins/load/load_fbx.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ class FbxLoader(load.LoaderPlugin):
 
     order = -10
 
-    families = ["staticMesh", "fbx"]
+    families = ["model", "staticMesh", "fbx"]
     representations = ["fbx"]
```
Comment (BigRoy) [inline openpype/hosts/houdini/plugins/load/load_fbx.py:20]: Another approach could be doing:
```suggestion
    families = ["*"]
    representations = ["*"]
    extensions = {"fbx"}
```
E.g. like this: https://github.com/ynput/OpenPype/blob/277a0baa7bbe89b2a58fbe9568271e4d5f72e86b/openpype/hosts/fusion/plugins/load/load_usd.py#L19-L21

Anyway, that's usually the way to go if the loader is supposed to be able to just load any FBX - no matter the family.

**Review by BigRoy — APPROVED** [review]
Looks good to me.

File: openpype/hosts/houdini/plugins/load/load_fbx.py (hunk 1 above)
Comment (MustafaJafar) [reply openpype/hosts/houdini/plugins/load/load_fbx.py:20]: so cool!

**Review by moonyuet — APPROVED** [review]
Works well to import fbx meshes published from other DCCs.
image

**Review by moonyuet — APPROVED** [review]
Looks Good....
image

**Pull request merged**